index.html

```diff
@@ -174,95 +182,90 @@ <h3>
           </li>
           <li>
             <a href="https://www.medium.com" target="_self"
-              ><img src="medium.png" alt="medium icon"
+              ><img src="images/medium.png" alt="medium icon"
             /></a>
           </li>
           <li>
             <a href="https://www.github.com" target="_self"
-              ><img src="github.png" alt="github icon"
+              ><img src="images/github.png" alt="github icon"
             /></a>
           </li>
           <li>
             <a href="https://www.vectorstock.com" target="_self"
-              ><img src="v.png" alt="vector stock icon"
+              ><img src="images/v.png" alt="vector stock icon"
             /></a>
           </li>
         </ul>
         <button type="button">Get my resume</button>
         <article class="languages_container">
           <div class="lang tags">
             <h3>Languages</h3>
-            <img class="downBTN" src="downBTN.png" alt="togbtn" />
+            <img class="downBTN" src="images/downBTN.png" alt="togbtn" />
           </div>
           <ul class="toggle">
             <li class="lan">
-              <img src="js.png" alt="js icon" />
+              <img src="images/js.png" alt="js icon" />
               <p>JavaScript</p>
             </li>
             <li class="lan">
-              <img src="html.png" alt="html icon" />
+              <img src="images/html.png" alt="html icon" />
               <p>HTML</p>
             </li>
             <li class="lan">
-              <img src="css.png" alt="css icon" />
+              <img src="images/css.png" alt="css icon" />
               <p>CSS</p>
             </li>
           </ul>
           <div class="lang">
             <h3>Frameworks</h3>
-            <img class="downBTN" src="rightBTN.png" alt="togbtn" />
+            <img class="downBTN" src="images/rightBTN.png" alt="togbtn" />
           </div>
           <div class="lang">
             <h3>Skills</h3>
-            <img class="downBTN" src="rightBTN.png" alt="togbtn" />
+            <img class="downBTN" src="images/rightBTN.png" alt="togbtn" />
           </div>
         </article>
       </section>
       <section class="contactForm">
         <div class="txt">
           <h2>Contact me</h2>
           <p>
-            If you have an application you are interested in developing, a feature
-            that you need built or a project that needs coding. I’d love to help
-            with it!
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
           </p>
         </div>
         <form action="https://formspree.io/f/mknyobde" method="post">
           <input
             type="text"
             name="name"
             id="name"
-            minlength="30"
-            placeholder="Lorem Ipsum"
+            max-length="30"
+            placeholder="Your name (first and last)"
```

Comment:
Nice job integrating the `formspre` service in your form😉. Let's try to make our project match the given design😉. Kindly take note of the spacings (margin and padding) around each element and try to make it match the design😉.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="400" alt="image" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/64643995/ad3cb0da-ac6b-4dd0-9ae6-965e64713324">|<img width="400" alt="image" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/64643995/7edbd859-f7a2-48d0-bde0-d52157120b45">|

Also, Kindly ensure the background image is completely present in the contact form  and add some paddings to your button to make it match the design😉.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="388" alt="image" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/64643995/2a2552a0-743c-4e3d-aa12-01eb1d3704ba">|<img width="417" alt="image" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/64643995/8231f6ea-60fc-4a9b-b58f-06696141fa0b">|